Index: gdata/inst/unitTests/runit.trim.R
===================================================================
--- gdata/inst/unitTests/runit.trim.R	(revision 981)
+++ gdata/inst/unitTests/runit.trim.R	(working copy)
@@ -2,7 +2,7 @@
 ###------------------------------------------------------------------------
 ### What: Tests for trim
 ### $Id$
-### Time-stamp: <2006-08-29 14:21:02 ggorjan>
+### Time-stamp: <2006-09-18 13:13:56 ggorjan>
 ###------------------------------------------------------------------------
 
 ### {{{ --- Test setup ---
@@ -20,16 +20,23 @@
   sTrim <- "    this is an example string    "
   sTrimR <- "this is an example string"
 
-  fTrim <- c(sTrim, sTrim, " A", " B ", "  C ", "D ")
-  fTrimR <- c(sTrimR, sTrimR, "A", "B", "C", "D")
+  cTrim <- c(sTrim, sTrim, " A", " B ", "  C ", "D ")
+  cTrimR <- c(sTrimR, sTrimR, "A", "B", "C", "D")
 
-  lTrim <- list(s=rep(sTrim, times=6), f=fTrim, i=1:6)
-  lTrimR <- list(s=rep(sTrimR, times=6), f=fTrimR, i=1:6)
+  fTrim <- factor(cTrim)
+  fTrimR <- factor(cTrimR)
 
+  lTrim <- list(c=cTrim, f=fTrim, i=1:6)
+  lTrimR <- list(c=cTrimR, f=fTrimR, i=1:6)
+
   dfTrim <- as.data.frame(lTrim)
   dfTrimR <- as.data.frame(lTrimR)
+  ## as characters are converted to factors by default
+  dfTrim$c <- as.character(dfTrim$c)
+  dfTrimR$c <- as.character(dfTrimR$c)
 
   checkIdentical(trim(sTrim), sTrimR)
+  checkIdentical(trim(cTrim), cTrimR)
   checkIdentical(trim(fTrim), fTrimR)
   checkIdentical(trim(lTrim), lTrimR)
   checkIdentical(trim(dfTrim), dfTrimR)
