Index: gdata/man/trim.Rd
===================================================================
--- gdata/man/trim.Rd	(revision 981)
+++ gdata/man/trim.Rd	(working copy)
@@ -2,13 +2,14 @@
 \alias{trim}
 \title{Remove leading and trailing spaces from character strings}
 \description{
- Remove leading and traling spaces from character strings
+ Remove leading and trailing spaces from character strings
 }
 \usage{
-trim(s)
+trim(s, sort=TRUE)
 }
 \arguments{
   \item{s}{object to be processed}
+  \item{sort}{logical, only for factors, look into details}
 }
 
 \details{
@@ -18,12 +19,18 @@
 factor \code{s} trims \code{\link{levels}}. There are also methods for
 \code{list} and \code{data.frame}.
 
+Trimming the levels of a factor can change the order of levels in some
+locales and levels are by default reordered. If this is not desired,
+\code{sort=FALSE} should be used.
+
 }
 \value{
-  \code{s} with all leading and traling spaces removed in its elements.
+  \code{s} with all leading and trailing spaces removed in its elements.
 }
-\author{ Gregory R. Warnes \email{warnes@bst.rochester.edu} }
-\seealso{ \code{\link[base]{sub}}, \code{\link[base]{gsub}} }
+\author{ Gregory R. Warnes \email{warnes@bst.rochester.edu}  with
+  contributions by Gregor Gorjanc}
+\seealso{ \code{\link[base]{sub}}, \code{\link[base]{gsub}} as well as
+  argument \code{strip.white} in \code{\link{read.table}}}
 \examples{
 s <- "    this is an example string    "
 trim(s)
@@ -38,4 +45,5 @@
 trim(df)
 
 }
+\keyword{manip}
 \keyword{character}
